tests/integration/scenarios/dash_live.js

```diff
@@ -108,6 +108,7 @@ describe("dash live SegmentTimeline content", function () {
     expect(typeof videoRepresentation.index).to.equal("object");
 
     const videoRepresentationIndex = videoRepresentation.index;
+    const initVideoSegment = videoRepresentationIndex.getInitSegment();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "initVideoSegment" ([javascript:S1854](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=249&issues=AWQT3116YwZU9vvrYAol&open=AWQT3116YwZU9vvrYAol)

[ISSUE_KEY]: AWQT3116YwZU9vvrYAol